src/controllers/image-request.ts

```diff
@@ -1,30 +1,62 @@
 import { CloudFrontRequest } from 'aws-lambda';
 import { imageService } from '@services';
 import { logger } from '../common/logger';
+import { ImageTypes } from '../common/constants';
 
-const canAcceptAvif = (
+const bestAcceptedFormat = (
   acceptHeader:
     | Array<{
         key?: string | undefined;
         value: string;
       }>
     | undefined,
-): boolean => {
+): ImageTypes => {
   if (acceptHeader === undefined) {
-    return false;
+    logger.info({
+      message: 'no accept header',
+    });
+    return ImageTypes.webp;
   }
 
+  // get all the header values out incase there is multiple 'accept' headers
+  const headerValues: string[] = [];
```

Comment:
would a `.reduce` make this clearer/ more concise?

Reply:
not really, we still need to split out each header value incase there is multiple accept headers passed and also catering for each one being able to have multiple accepts in it

Reply:
`.includes` will behave the same in a single string or in an array

Reply:
In this implementation

Reply:
alternatively if you want to keep it as an array the outcome can be achieved by doing this:
```
headerValues = acceptHeader.reduce(
  (previousValue, { value }) => previousValue + value,
  ''
).split(',')
```